core/src/main/java/org/elasticsearch/search/suggest/SuggestBuilder.java

```diff
@@ -98,6 +98,15 @@ public SuggestBuilder addSuggestion(SuggestionBuilder<?> suggestion) {
         return suggestions;
     }
 
+    /**
+     * Returns the possibly null global suggest text that
+     * should be applied as the text for all suggesters.
+     */
+    @Nullable
+    public String getGlobalText() {
```

Comment:
Which name do you think is better, especially in light of it being an API change?  I thought `globalText` is more clear than `text`, but is it worth the API change, especially when the actual field name in the JSON is just `text`?